src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -1276,18 +1279,19 @@ private void succeedingTestForNominalVoltages(UUID networkUuid, String variantId
 
     @Test
     public void shouldReturnSubstationsTabData() throws Exception {
+        succeedingTestForElementsInfos(NETWORK_UUID, null, ElementType.SUBSTATION, InfoType.TAB, null, resourceToString("/substations-tab-data.json"));
```

Comment:
Why ? it's exactly the same line than 1283

Reply:
yes its duplicated, removed